src/bug-filing/common/markup/html-factory.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { escape } from 'lodash';
+import { MarkupFactory } from './markup-factory';
+
+const createFactory = (): MarkupFactory => {
+    const bold = (text: string): string => {
+        return `<b>${text}</b>`;
+    };
+
+    const snippet = (text: string): string => {
+        const maxSnippetLength = 256;
+        let constrainedSnippet = text;
+
+        if (text.length > maxSnippetLength) {
+            constrainedSnippet = text.substr(0, maxSnippetLength) + '...';
+        }
+
+        return `<code>${escape(constrainedSnippet)}</code>`;
```

Comment:
>escape [](start = 24, length = 6)

It feels off that snippet and howToFix in this factory use escape, but none of the other functions in either factory perform escaping. Should we be consistent?

Reply:
how to fix and snippet are the only places we expect to have html code we may need to scape, so not too sure about the consistency point here.